ui/packages/catalog-ui-search/src/main/webapp/component/input/input.view.js

```diff
@@ -60,6 +60,15 @@ define([
         },
         listenForChange: function(){
             this.$el.on('change keyup input', function(){
+                // only applies to anyText filters
+                if (this.$el.attr('data-id') === 'anyText') {
+                    if (this.$el.find('input').val() === '') {
+                        this.$el.addClass('has-warnings');
+                    } else {
+                        this.$el.removeClass('has-warnings');
+                    }
```

Comment:
❓Are we okay highlight the box with a warning but having no explanation of what is causing the warning? Maybe this case is obvious. Would it be easy/worthwhile to add a note when you hover over the box when it has a warning? Or add a note in the help view?

Reply:
I could add a text note that appears on hover that says something like "Text input is blank"

Reply:
🤷‍♀️ I defer to the @codice/ui reviewers to decide if that is necessary.